docs/documentation/_data/supported_versions.yml

```diff
@@ -21,7 +21,7 @@ osDistributions:
   ubuntu:
     name: Ubuntu
     url: https://ubuntu.com/download/server
-  '_redos':
+  redos:
```

Comment:
```suggestion
  '_redos'::
```
It was a dirty trick to proper items sorting  :)

Reply:
@borg-z JFI

Reply:
I see, I thought it was some kind of typo.